sporkfish/searcher/move_ordering/move_order_config.py

```diff
@@ -0,0 +1,39 @@
+from ...configurable import Configurable
+
+from enum import Enum
+
+
+class MoveOrderMode(Enum):
+    MVV_LVA = "MVV_LVA"
+    KILLER_MOVE = "KILLER_MOVE"
+    COMPOSITE = "COMPOSITE"
+
+
+class MoveOrderConfig(Configurable):
+    """
+    Configuration class for move ordering.
+    """
+
+    def __init__(
+        self,
+        move_order_mode: MoveOrderMode = MoveOrderMode.COMPOSITE,
+        mvv_lva_weight: float = 2.0,
+        killer_moves_weight: float = 1.0,
+    ):
+        """
+        Initializes MoveOrderConfig with specified parameters.
+
+        :param move_order_mode: The mode for move ordering. Default is MoveOrderMode.COMPOSITE.
+        :type move_order_mode: MoveOrderMode, optional
+        :param mvv_lva_weight: The weight for Most Valuable Victim/Least Valuable Aggressor heuristic. Default is 2.0.
+        :type mvv_lva_weight: float, optional
+        :param killer_moves_weight: The weight for killer moves heuristic. Default is 1.0.
+        :type killer_moves_weight: float, optional
```

Comment:
Sorry naive q - why is the `killer_moves_weight` type optional? What happens if we pass in `None`?

Reply:
It's optional in the sense that there's a default, ChatGPT's logic. I missed this - should change next PR TY